examples/fft_physical_x/fft_c2c_x.f90

```diff
@@ -129,12 +129,12 @@ program fft_c2c_x
    ierror = cudaDeviceSynchronize()
 #endif
 
-   call MPI_ALLREDUCE(t2, t1, 1, real_type, MPI_SUM, &
+   call MPI_ALLREDUCE(t2, t1, 1, MPI_DOUBLE, MPI_SUM, &
```

Comment:
To be absolutely safe this should be `MPI_DOUBLE_PRECISION` I think, and similar below and in other fft examples